Libs/Core/ctkJobScheduler.cpp

```diff
@@ -193,15 +190,13 @@ bool ctkJobSchedulerPrivate::insertJob(QSharedPointer<ctkAbstractJob> job)
   }
 
   emit q->jobInitialized(job->toVariant());
-  emit this->queueJobsInThreadPool();
+  this->onQueueJobsInThreadPool();
```

Comment:
Is it important to call the method directly instead of emitting a signal? If yes then add a comment to describe why.